src/renderer/widgets/DelegationModal/components/DelegationList.tsx

```diff
@@ -77,6 +78,8 @@ export const DelegationList = ({ onClick, onAddCustomClick }: Props) => {
             )}
           </div>
 
+          <AddToRegistry className="mx-5 mb-6" />
```

Comment:
You can simply wrap this component with div and revert all changes related to Alert

Reply:
In that case, I will need to duplication check does we closed message already or not in DelegationList component